tests/realtikvtest/brietest/brie_test.go

```diff
@@ -126,3 +128,21 @@ func TestCancel(t *testing.T) {
 		req.FailNow("the backup job doesn't be canceled")
 	}
 }
+
+func TestExistedTables(t *testing.T) {
```

Comment:
please add scenarios with multiple dup tables and also restore with filter.

Reply:
We can't customize filter on sql client now.